src/components/fields/ColorField.vue

```diff
@@ -141,14 +146,57 @@ export default {
       }, 500)
     },
     onColorChange(color) {
+      this.validateColor()
       this.checkTransparency()
 
-      if (color === this.valueToShow) {
+      if (color === this.valueToShow || !this.isValid) {
         return
       }
 
       sendCssToFrame(color, this.data.fieldConfig)
     },
+    validateColor() {
+      let errorsFields = document.querySelectorAll('input.error')
```

Comment:
you don't seem to be reassigning this value so this should be a `const`

Reply:
@squallstar fixed